src/main/java/com/czertainly/core/service/cmp/util/AlgorithmHelper.java

```diff
@@ -0,0 +1,31 @@
+package com.czertainly.core.service.cmp.util;
+
+import org.bouncycastle.asn1.x509.AlgorithmIdentifier;
+
+/**
+ * Helper to translate ANS1 -> JAVA algorithm names
+ */
+public class AlgorithmHelper {

```

Comment:
This should be removed and BouncyCastle used instead. We also have `AlgorithmUtil` in `com.czertainly.core.util` and from that perspective, this class seems to be duplicate and not necessary.

Reply:
DONE, need modified existing AlgorithmUtil.